<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="listWidget_contacts"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addContact">
             <property name="text">
              <string>Добавить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="delContact">
             <property name="text">
              <string>Удалить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListWidget" name="listWidget_message"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="lineEdit_message"/>
           </item>
           <item>
            <widget class="QPushButton" name="sendButton">
             <property name="text">
              <string>Отправить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonUp">
             <property name="text">
              <string>Обновить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QListWidget" name="console"/>
     </widget>
    </item>
   </layout>
   <zorder>listView_contacts</zorder>
   <zorder>listView_message</zorder>
   <zorder>lineEdit_message</zorder>
   <zorder>sendButton</zorder>
   <zorder></zorder>
   <zorder>splitter</zorder>
   <zorder>addContact</zorder>
   <zorder>delContact</zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder>console</zorder>
   <zorder>listWidget_message</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="login"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="login">
   <property name="text">
    <string>Вход</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Регистрация</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
